ports/meta/macathena-extra-software/Portfile

```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+PortSystem          1.0
+PortGroup		config_package 1.0
+
+name                macathena-extra-software
```

Comment:
We should probably install X itself, since Macs don't come with that by default. :)

(`xorg-server`)